README.md

```diff
@@ -54,11 +54,11 @@ Later:
 ## :rocket: Getting started
 
 The quickest way to get a `hydra-node` running is to use our [docker
-images](https://hub.docker.com/r/inputoutput/hydra/tags).
+images](https://hub.docker.com/r/inputoutput/hydra-node/tags).
 
 ```sh
-docker pull inputoutput/hydra:hydra-node-latest
-docker run --rm inputoutput/hydra:hydra-node-latest --help
+docker pull inputoutput/hydra-node
```

Comment:
Question: Is ghcr.io always resolved as a repository, even when you never used it before?

Reply:
Yes.